packages/zksync-dai-bridge/contracts/l2/dai.sol

```diff
@@ -202,16 +204,16 @@ contract Dai {
   }
 
   // --- Approve by signature ---
+
   function permit(
     address owner,
     address spender,
     uint256 value,
     uint256 deadline,
-    uint8 v,
-    bytes32 r,
-    bytes32 s
-  ) external {
+    bytes memory signature
+  ) public {
     require(block.timestamp <= deadline, "Dai/permit-expired");
+    require(owner != address(0), "Dai/invalid-permit");
```

Comment:
Maybe this error message could be changed now?

Reply:
@olivdb @luke7211 do we intend to change this?